<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConstraintBaseDialog_template</class>
 <widget class="QDialog" name="ConstraintBaseDialog_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="instructionsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Instruction text</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="leftVerticalLayout">
       <item>
        <widget class="QLabel" name="constraintsTextLabel">
         <property name="text">
          <string>Constraints</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="constraintsListWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightVerticalLayout">
       <item>
        <widget class="QLabel" name="currentConstraintTextLabel">
         <property name="text">
          <string>Current constraint</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="currentConstraintTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <item>
      <widget class="QPushButton" name="addConstraintPushButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyConstraintPushButton">
       <property name="text">
        <string>Modify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeConstraintPushButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>constraintsListWidget</tabstop>
  <tabstop>currentConstraintTextEdit</tabstop>
  <tabstop>addConstraintPushButton</tabstop>
  <tabstop>modifyConstraintPushButton</tabstop>
  <tabstop>removeConstraintPushButton</tabstop>
  <tabstop>helpPushButton</tabstop>
  <tabstop>closePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
